### PR TITLE
Tink#238 added new method to fetch workflow context list

### PR DESCRIPTION
**Pull request by Cbkhare**
With recent changes in `GetWorkflowContexts` method as per the commit https://github.com/tinkerbell/tink/commit/46881647de83f85ea00b2e51385b9be5e10f470e#diff-a223963715d7c23da698d5963075810eL36 boots does not get the intended list of workflows in the tink server. The worker is not able to PXE boot. Boots needs modification as per https://github.com/tinkerbell/tink/pull/237 to complete. 

## Expected Behaviour
Boots should get the correct list of workflowcontext lists from tink. 

## Current Behaviour
pxe-fail

Boots logs:
```
{"level":"info","ts":1596648510.5594132,"caller":"src/tftp.go:60","msg":"","service":"github.com/tinkerbell/boots","pkg":"main","client":"192.168.1.5","event":"open","filename":"undionly.kpxe","error":"retrieved job is empty: fetching the workflow","errorVerbose":"fetching the workflow\ngithub.com/tinkerbell/boots/packet.(*Client).GetWorkflowsFromTink\n\t/drone/src/packet/endpoints.go:54\ngithub.com/tinkerbell/boots/job.CreateFromIP\n\t/drone/src/job/job.go:106\nmain.tftpHandler.ReadFile\n\t/drone/src/tftp.go:51\ngithub.com/betawaffle/tftp-go.(*session).serveRRQ\n\t/go/pkg/mod/github.com/betawaffle/tftp-go@v0.0.0-20160921192434-dc649c1318ff/handler.go:228\ngithub.com/betawaffle/tftp-go.(*session).serve\n\t/go/pkg/mod/github.com/betawaffle/tftp-go@v0.0.0-20160921192434-dc649c1318ff/handler.go:158\ngithub.com/betawaffle/tftp-go.serve\n\t/go/pkg/mod/github.com/betawaffle/tftp-go@v0.0.0-20160921192434-dc649c1318ff/handler.go:95\ngithub.com/betawaffle/tftp-go.Serve.func1\n\t/go/pkg/mod/github.com/betawaffle/tftp-go@v0.0.0-20160921192434-dc649c1318ff/server.go:164\nruntime.goexit\n\t/usr/local/go/src/runtime/asm_amd64.s:1357\nretrieved job is empty"}
{"level":"info","ts":1596648510.5598457,"caller":"src/tftp.go:70","msg":"","service":"github.com/tinkerbell/boots","pkg":"main","client":"192.168.1.5","event":"open","filename":"undionly.kpxe","error":"access_violation: permission denied","errorVerbose":"permission denied\naccess_violation\nmain.tftpHandler.ReadFile\n\t/drone/src/tftp.go:70\ngithub.com/betawaffle/tftp-go.(*session).serveRRQ\n\t/go/pkg/mod/github.com/betawaffle/tftp-go@v0.0.0-20160921192434-dc649c1318ff/handler.go:228\ngithub.com/betawaffle/tftp-go.(*session).serve\n\t/go/pkg/mod/github.com/betawaffle/tftp-go@v0.0.0-20160921192434-dc649c1318ff/handler.go:158\ngithub.com/betawaffle/tftp-go.serve\n\t/go/pkg/mod/github.com/betawaffle/tftp-go@v0.0.0-20160921192434-dc649c1318ff/handler.go:95\ngithub.com/betawaffle/tftp-go.Serve.func1\n\t/go/pkg/mod/github.com/betawaffle/tftp-go@v0.0.0-20160921192434-dc649c1318ff/server.go:164\nruntime.goexit\n\t/usr/local/go/src/runtime/asm_amd64.s:1357"}
```

## Possible Solution
This could be due to the changes made for issue #222.
## Steps to Reproduce (for bugs)
1. Setup locally with Vagrant
2. Define a workflow (say Hello-World)
3. Boot the worker machine
4. Observe logs in Boots

## Your Environment

* Operating System and version? : Linux (Ubuntu 18.04)

* How are you running Tinkerbell? : Using Vagrant & VirtualBox

### CODE REVIEW

File: go.sum [hunk 1]
```diff
@@ -43,6 +42,8 @@ github.com/gammazero/deque v0.0.0-20200227231300-1e9af0e52b46 h1:iX4+rD9Fjdx8Skm
 github.com/gammazero/deque v0.0.0-20200227231300-1e9af0e52b46/go.mod h1:D90+MBHVc9Sk1lJAbEVgws0eYEurY4mv2TDso3Nxh3w=
 github.com/gammazero/workerpool v0.0.0-20200311205957-7b00833861c6 h1:1Cy/haf7XO4OyrkGid0Wq5CMluIErbvDptVAt8UTy38=
 github.com/gammazero/workerpool v0.0.0-20200311205957-7b00833861c6/go.mod h1:/XWO2YAUUpPi3smDlFBl0vpX0JHwUomDM/oRMwRmnSs=
+github.com/gauravgahlot/tink v0.0.0-20200806080035-1ab6de4bab5c h1:lJ8Tl+jk1pvzmB+GQKUJHcFPswUNLEARqk4JunWtXg8=
+github.com/gauravgahlot/tink v0.0.0-20200806080035-1ab6de4bab5c/go.mod h1:5Kv1re+s/uSfE3vWsoOS+tKwZYwLrrPXgEXViTbEmv4=
```
Comment (gauravgahlot) [inline go.sum:46]: I believe this will removed once we have my PR merged. :thinking:

Comment (Cbkhare) [reply go.sum:46]: yes, let's have your tink PR merged, then e can update this.

**Comment by gianarb** [comment]
You have to fix gomod first

**Comment by Cbkhare** [comment]
> You have to fix gomod first

done

**Pull request merged**